src/utils/navigation.ts

```diff
@@ -86,5 +86,10 @@ export const NAVIGATION: Navigation = {
     currentPage: config.MANAGING_OFFICER_CORPORATE_PAGE,
     previousPage: config.BENEFICIAL_OWNER_TYPE_URL,
     nextPage: config.BENEFICIAL_OWNER_TYPE_URL
+  },
+  [config.TRUST_INFO_URL]: {
+    currentPage: config.TRUST_INFO_PAGE,
+    previousPage: config.BENEFICIAL_OWNER_TYPE_URL,
+    nextPage: config.CHECK_YOUR_ANSWERS_URL
```

Comment:
Just wondering if you need to change this section above:

```
[config.BENEFICIAL_OWNER_TYPE_URL]: {
    currentPage: config.BENEFICIAL_OWNER_TYPE_PAGE,
    previousPage: config.BENEFICIAL_OWNER_STATEMENTS_URL,
    nextPage: config.CHECK_YOUR_ANSWERS_URL
  },
```

Or does it stay as is since trust data may or may not be present.

Reply:
No response to this comment so not sure if you agree and a change was made or was missed (generally useful to respond to all comments so it's clear that they've been read). Thanks.

Reply:
I think the 
nextPage: config.CHECK_YOUR_ANSWERS_URL
will have to change to be an array as it can go to 2 next pages depending on trusts or not
i would change it to 
`nextPage: [
   config.CHECK_YOUR_ANSWERS_URL,
   config.TRUST_INFO_PAGE ]`

I don't think we currently use the next page value but plan to in future. Perhaps @mouhajer-ch can confirm what I've written is correct

Reply:
I think the check_your_answers_page previousPage would have to be an array as well as could have come from either trusts page and beneficial_owner_type page

Reply:
👍 not ideal that we have code in there for the future, that we're not using yet. Should be added at the point it's used.

Reply:
That doesn't seem to work, @markpit because the Navigation interface type expects a `string` not an `array`

Reply:
Sounds like this is fine for now as nextPage not currently used but when it is, this may not work. Would suggest adding a comment where the array list would normally be defined, just to point out that what's currently configured is not entirely correct.